wolfcrypt/test/test.c

```diff
@@ -1640,11 +1641,20 @@ options: [-s max_relative_stack_bytes] [-m max_relative_heap_memory_bytes]\n\
         printf("Testing complete. You may close the window now\n");
         while (1);
 #endif
-#ifndef WOLFSSL_ESPIDF
+
+#ifdef WOLFSSL_ESPIDF
+        ESP_LOGI("wolfcrypt_test", "Exiting main with return code: % d\n", args.return_code);
+        return args.return_code;
```

Comment:
Both cases return `return args.return_code;`. Can you move that below these macros?

Reply:
ah yes, nice cleanup suggestion. done in latest commit.